sysutils/salt/Portfile

```diff
@@ -1,60 +1,111 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
-PortSystem          1.0
-
-name                salt
-version             2017.5
-revision            1
-categories          sysutils
-platforms           darwin
-maintainers         gmail.com:jeremy.mcmillan
-license             Apache-2
-supported_archs     noarch
-description         Salt is a Python-based remote execution, automation, \
-                    configuration, and orchestration engine.
-long_description    SaltStack is fast, scalable and flexible software for data \
-                    center automation, from infrastructure and any cloud, \
-                    to the entire application stack.
-homepage            http://saltstack.com/
+PortSystem		1.0
```

Comment:
You're still using tabs.

Reply:
OK. This is weird. On my local clone, the file has all tabs expanded to spaces, but in the GitHub file editor web UI it has tabs.

Reply:
Did you forget to use `git add` before committing?

Reply:
I forgot to use git add before committing.

Thanks @l2dy @raimue.